Changed format_version default value to 4 (#8662)
pingcap/docs: tiflash/tiflash-configuration.md

    ## DTFile format
    ## * format_version = 1, the old format, deprecated.
    ## * format_version = 2, the default format for versions < v6.0.0.

>     ## * format_version = 3, the default format for versions v6.0.0, which provides more data validation features.

Reviewer: ```suggestion
    ## * format_version = 3, the default format for v6.0.0, which provides more data validation features.
```

>     ## * format_version = 4, the default format for versions >= v6.1.0, which provides lower write amplification and lower background task resource consumption

Reviewer: ```suggestion
    ## * format_version = 4, the default format for v6.1.0 and later versions, which provides lower write amplification and lower background task resource consumption
```

    ## * format_version = 1, the old format, deprecated.
    ## * format_version = 2, the default format for versions < v6.0.0.

>     ## * format_version = 3, the default format for versions >= v6.0.0, which provides more data validation features.

Reviewer 2: ```suggestion
    ## * format_version = 3, the default format for versions v6.0.0, which provides more data validation features.
```

>     ## * format_version = 4, the default format for versions >= v6.1.0, which provides PageFormat::V3. PageFormat::V3 is compatible with previous PageFormat versions

Reviewer 2: ```suggestion
    ## * format_version = 4, the default format for versions >= v6.1.0, which provides lower write amplification and lower background task resource consumption
```
